modoptions.lua

```diff
@@ -1267,6 +1228,7 @@ local options = {
         name   	= "No Rush Time",
         desc   	= "(Range: 0 - 30). Minutes",
         type   	= "number",
+        hidden 	= true,
```

Comment:
rush timer was an experimental modoption considered for inclusion?

Reply:
It was, it still is, but it's too unfinished to be put into restrictions, and it wasn't touched for ages.